src/pybind/mgr/calamari_rest/manager/user_request.py

```diff
@@ -0,0 +1,553 @@
+import json
+import logging
+import uuid
+
+from calamari_rest.types import OsdMap, PgSummary, USER_REQUEST_COMPLETE, USER_REQUEST_SUBMITTED
+from calamari_rest.util import now
+from mgr_module import CommandResult
+
+from rest import logger
+log = logger()
```

Comment:
this form of instantiating the logger instance will use the `root` logger. The caveat of doing this is that any log statements in this file will *not* have the ability to:

* show where the log statement is originating from (e.g. `calamari_rest.manager.user_request`)
* be configured differently than the default root logger